lib/mutant/mutator/node/binary.rb

```diff
@@ -60,6 +61,21 @@ def emit_left_negation
           emit(s(node.type, n_not(left), right))
         end
 
+        # Emit mutation from `a[b] || c` to `a.fetch(b) { c }`
+        #
+        # @return [undefined]
+        def emit_fetch_mutation
+          send_node = AST::Meta::Send.new(left)
```

Comment:
@mbj I ended up adding a test case that used `csend` which justifies including `n_send?`